src/components/ExampleListComponent/RenderExampleList.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const RenderExampleList = (props) => (
+  <div>
+    {
+      props.data.map(item => (
+        <figure key={item.id}>
+          <img src={item.thumbnailUrl} alt={item.title} />
+          <figcaption>
+            <h3>{item.title}</h3>
+          </figcaption>
+        </figure>
+      ))
+    }
+  </div>
+)
+
+export default RenderExampleList;
+
+RenderExampleList.propTypes = {
+  data: PropTypes.arrayOf(
+    PropTypes.shape({
```

Comment:
This is huge! Giving the students a lesson on `PropTypes` and `type checking`, in general, will go a LONG way IMO